### ISSUE TITLE
Add sound based on sound design

### ISSUE THREAD

**Issue by jbphet** [phetsims/john-travoltage#294]
An initial sound design for this simulation has been reviewed and approved.  This isn't necessarily final, but it is the starting point for adding sound to this simulation.  The prototype version is available at https://bayes.colorado.edu/dev/html/phet-io-wrapper-sonification/1.0.0-dev.69/phet-io-wrapper-sonification/john-travoltage/html/john-travoltage-sonification.html?sonificationFile=sonificationMix&accessibility.

This version should use the recently developed sound library from the tambo repo.

**Comment by jbphet** [comment]
https://phet-dev.colorado.edu/html/john-travoltage/1.4.0-dev.9/phet/john-travoltage_en_phet.html?supportsSound&supportsEnhancedSound

**Comment by jbphet** [comment]
Comments from @Ashton-Morris based on v1.5.0-dev.2:

```
    soundManager.setReverbLevel(0.02);
    var ouchAudioPlayer = new OneShotSoundClip( ouchAudio, { initialOutputLevel: 0.7 } );
    soundManager.addSoundGenerator( ouchAudioPlayer );
    var gazouchAudioPlayer = new OneShotSoundClip( gazouchAudio, { initialOutputLevel: 0.8 } );
    soundManager.addSoundGenerator( gazouchAudioPlayer );
    var electricDischargeAudioPlayer = new OneShotSoundClip( electricDischargeAudio, { initialOutputLevel: 0.75 } );
    soundManager.addSoundGenerator( electricDischargeAudioPlayer );
    var chargesInBodyAudioPlayer = new LoopingSoundClip( chargesInBodyAudio, {
      autoDetectLoopBounds: true,
      initialOutputLevel: 0.1
    } );
    soundManager.addSoundGenerator( chargesInBodyAudioPlayer );
    soundManager.addSoundGenerator( new ArmPositionSoundGenerator( model.arm.angleProperty, {
      enableControlProperties: [ resetNotInProgressProperty ],
      initialOutputLevel: 0.2
    } ) );
    this.footDragSoundGenerator = new FootDragSoundGenerator( model.leg.angleProperty, {
      enableControlProperties: [ resetNotInProgressProperty ],
      initialOutputLevel: 0.7
    } );
    soundManager.addSoundGenerator( this.footDragSoundGenerator );
    var popSoundGenerator = new PitchedPopGenerator( {
      enableControlProperties: [ resetNotInProgressProperty ],
      initialOutputLevel: 1
    } );
```

And here is the charges in body sound level.

```
  var CHARGES_SOUND_GAIN_FACTOR = 0.1; // multiplier for charges-in-the-body sound, empicially determined
```

**Comment by jbphet** [comment]
A dev version with the adjustments described in the comment above has been deployed, see https://phet-dev.colorado.edu/html/john-travoltage/1.5.0-dev.3/phet/john-travoltage_en_phet.html?supportsSound&supportsEnhancedSound.

I realized while implementing these that I didn't let @Ashton-Morris know about setting the level for the Reset All sound, so I took a shot at it, trying to balance it with the other sound levels.  I ended up setting the initial output level to 0.7.
